### DISCUSSION TITLE
How to switch profiles between projects?

### DISCUSSION THREAD

**Discussion by cristianslv** [alefragnani/vscode-project-manager#755] [Q&A] [answered]
There was a commit in the project this year https://github.com/alefragnani/vscode-project-manager/commit/1f4c0eccbfb8b01b9b01a4457d5767fc51829b43

But I still can't change profiles between projects, is that possible?

**Comment by alefragnani** [comment, marked as answer]
Hi @cristianslv ,

Yes, it was merged, but the version was not released yet. 

I'm planning to release it in the first quarter of September, if everything goes as planned.

Hope this helps
